features/support/utils.rb

```diff
@@ -2,11 +2,7 @@ module StepUtils
   def wait_for_client_page_refresh
     page.driver.execute_script("window.__page_refreshed = false; CMSimple.Page.one('refresh', function(){ window.__page_refreshed = true; });")
     yield
-    wait_until(10) do
-      page.evaluate_script('window.__page_refreshed') == true
-      # not sure why this makes the tests pass
-      sleep(1)
-    end
+    page.evaluate_script('window.__page_refreshed') == true
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.